website/docs/getting-started/setup/installation-guides/google-cloud-run/manage-traffic.mdx

```diff
@@ -0,0 +1,98 @@
+---
+description: Configure Google Cloud Run for supporting the High Availability and Scalability.
+toc_max_heading_level: 2
+hide_title: true
+title: Configure High Availability and Scalability
+---
+
+<!-- vale off -->
+
+<div className="tag-wrapper">
+ <h1>Configure High Availability and Scalability</h1>
+
+<Tags
+tags={[
+{ name: "Business", link: "https://www.appsmith.com/pricing", additionalClass: "business" }
+]}
+/>
+
+</div>
+
+<!-- vale on -->
+
+This page provides steps to configure Google Cloud Run Appsmith installation for high availability and scalability.
+
+## Prerequisites
+
+* A [Google Cloud](https://console.cloud.google.com/) account.
+* An Appsmith Commercial Edition installation on Google Cloud Run. If not installed yet, see the [Google Cloud Run installation guide](/getting-started/setup/installation-guides/google-cloud-run) for installing Appsmith.
+* If you want scalable, secure, and highly available services, enable [Memorystore for Redis API](https://console.cloud.google.com/apis/dashboard).
```

Comment:
now this is not optional in this dedicated page for scalability. 
```suggestion
* For scalable, secure, and highly available services, enable [Memorystore for Redis API](https://console.cloud.google.com/apis/dashboard).
```

Reply:
I'm not sure. @pratapaprasanna - Could you please confirm this?